convert2rhel/data/7/x86_64/configs/suse_es-7-x86_64.cfg

```diff
@@ -0,0 +1,12 @@
+[system_info]
+
+# SUSE ES packages are not signed
+gpg_fingerprints =
```

Comment:
This seems off, I don't think this is true...

Reply:
From what we know, it is true. Unless SUSE changed it in the recent year. Do you have access to [a RHEL system updated/patched through the SUSE Manager](https://www.suse.com/c/know-suse-supporting-rhat-years/) to verify that the packages provided by SUSE are really signed?

Reply:
I'm checking into this...

Reply:
Unfortunately, this is, in fact, true. The package builds are not done by SUSE, so they aren't signed with SUSE's key.